src/NuGet.Indexing/PackageAnalyzer.cs

```diff
@@ -15,17 +17,18 @@ static PackageAnalyzer()
         {
             _fieldAnalyzers = new Dictionary<string, Analyzer>(StringComparer.OrdinalIgnoreCase)
             {
-                { "Id", new IdentifierKeywordAnalyzer() },
-                { "IdAutocomplete", new IdentifierAutocompleteAnalyzer() },
-                { "TokenizedId", new IdentifierAnalyzer() },
-                { "ShingledId", new ShingledIdentifierAnalyzer() },
-                { "Version", new VersionAnalyzer() },
-                { "Title", new DescriptionAnalyzer() },
-                { "Description", new DescriptionAnalyzer() },
-                { "Summary", new DescriptionAnalyzer() },
-                { "Authors", new DescriptionAnalyzer() },
+                { LuceneConstants.IdPropertyName, new IdentifierKeywordAnalyzer() },
+                { LuceneConstants.IdAutocompletePropertyName, new IdentifierAutocompleteAnalyzer() },
+                { LuceneConstants.TokenizedIdPropertyName, new IdentifierAnalyzer() },
+                { LuceneConstants.ShingledIdPropertyName, new ShingledIdentifierAnalyzer() },
+                { LuceneConstants.NormalizedVersionPropertyName, new VersionAnalyzer(caseSensitive: true) },
+                { LuceneConstants.CaseInsensitiveNormalizedVersionPropertyName, new VersionAnalyzer(caseSensitive: false) },
+                { LuceneConstants.TitlePropertyName, new DescriptionAnalyzer() },
+                { LuceneConstants.DescriptionPropertyName, new DescriptionAnalyzer() },
+                { LuceneConstants.SummaryPropertyName, new DescriptionAnalyzer() },
+                { LuceneConstants.AuthorsPropertyName, new DescriptionAnalyzer() },
                 { "Owner", new OwnerAnalyzer() },
```

Comment:
Any reason for not adding a constant for `Owner`?

Reply:
I could not find any other places this was used (in the same way). Investigation is out of scope.

Reply:
Turns out it's unused. I removed it.